From dcfa22bd1d12fd521af98a7c38f4cebfed0eedc4 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <uwu@icenowy.me>
Date: Wed, 20 May 2026 23:28:15 +0800
Subject: [PATCH 256/344] RUYI: PCI: add SpacemiT vendor id and its K3 device
 id to pci_ids

The SpacemiT K3 chip's root complex needs to be listed in the allowlist
of rtw89 driver to allow 36-bit DMA.

Add the vendor and device IDs to pci_ids.h header file.

Signed-off-by: Icenowy Zheng <uwu@icenowy.me>
Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 include/linux/pci_ids.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index e312f1945f61a..67cf7e74eccf1 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2642,6 +2642,9 @@
 #define PCI_VENDOR_ID_SUNIX		0x1fd4
 #define PCI_DEVICE_ID_SUNIX_1999	0x1999
 
+#define PCI_VENDOR_ID_SPACEMIT		0x201f
+#define PCI_DEVICE_ID_SPACEMIT_K3	0x0002
+
 #define PCI_VENDOR_ID_HINT             0x3388
 #define PCI_DEVICE_ID_HINT_VXPROII_IDE 0x8013
 
-- 
2.52.0

